src/bodypart.cpp

```diff
@@ -65,7 +65,7 @@ std::string body_part_name (body_part bp)
     case bp_foot_r:
         return _("right foot");
     default:
-        return _("appendix");
+        return _("appendage");
```

Comment:
Want to open a separate issue for this? It's not directly related to this PR, and I'm not going to look through the code to verify it's not used because I haven't yet encountered any bionics and don't want to spoil it for my playthroughs.